pkg/platform/api/buildplanner/model/buildplan.go

```diff
@@ -266,39 +282,51 @@ func (b *BuildPlanByProject) CommitID() (strfmt.UUID, error) {
 }
 
 type BuildPlanByCommit struct {
-	Commit *Commit `json:"commit"`
+	commit *Commit `json:"commit"`
```

Comment:
I believe that by making this field private the `encoding/json` package will not be able to access it so marhsalling/unmarshalling may fail. Maybe that's changed but I remember getting caught by this before.

Reply:
Urgh you're right, thanks for catching!